python/tach/interactive/modules.py

```diff
@@ -119,14 +120,18 @@ def build_from_path(
         path: Path,
         depth: int | None = 1,
         exclude_paths: list[str] | None = None,
+        use_regex_matching: bool | None = None,
```

Comment:
```suggestion
        use_regex_matching: bool | None = False,
```
Let's explicitly default to False for now